tools/migration/ppaas-artifact-converter/ppaas-artifact-converter-log.log

```diff
@@ -0,0 +1,44 @@
+2015-11-30 23:18:18 ERROR TemplateLoader:145 - java.io.FileNotFoundException: /home/nishadi/ppaas-artifact-converter/main/resources/template/network-partitions/gce/network-partition-1.json (No such file or directory)
+2015-11-30 23:18:18 ERROR ArtifactConverter:39 - java.lang.ClassCastException: com.google.gson.internal.LinkedTreeMap cannot be cast to org.apache.stratos.rest.endpoint.bean.autoscaler.partition.Partition
+2015-11-30 23:20:40 ERROR ArtifactConverter:39 - java.lang.ClassCastException: com.google.gson.internal.LinkedTreeMap cannot be cast to org.apache.stratos.rest.endpoint.bean.autoscaler.partition.Partition
+2015-11-30 23:23:54 ERROR TemplateLoader:145 - java.io.FileNotFoundException: /home/nishadi/ppaas-artifact-convertersrc/main/resources/templates/network-partitions/gce/network-partition-1.json (No such file or directory)
+2015-11-30 23:23:54 ERROR ArtifactConverter:39 - java.lang.ClassCastException: com.google.gson.internal.LinkedTreeMap cannot be cast to org.apache.stratos.rest.endpoint.bean.autoscaler.partition.Partition
+2015-11-30 23:24:04 ERROR TemplateLoader:145 - java.io.FileNotFoundException: /home/nishadi/ppaas-artifact-converter/resources/templates/network-partitions/gce/network-partition-1.json (No such file or directory)
+2015-11-30 23:24:04 ERROR ArtifactConverter:39 - java.lang.ClassCastException: com.google.gson.internal.LinkedTreeMap cannot be cast to org.apache.stratos.rest.endpoint.bean.autoscaler.partition.Partition
+2015-11-30 23:24:42 ERROR TemplateLoader:145 - java.io.FileNotFoundException: /home/nishadi/ppaas-artifact-convertersrc/main/resources/templates/network-partitions/gce/network-partition-1.json (No such file or directory)
+2015-11-30 23:24:42 ERROR ArtifactConverter:39 - java.lang.ClassCastException: com.google.gson.internal.LinkedTreeMap cannot be cast to org.apache.stratos.rest.endpoint.bean.autoscaler.partition.Partition
+2015-11-30 23:25:17 ERROR ArtifactConverter:39 - java.lang.ClassCastException: com.google.gson.internal.LinkedTreeMap cannot be cast to org.apache.stratos.rest.endpoint.bean.autoscaler.partition.Partition
+2015-12-01 06:33:42 ERROR Transformation:157 - /home/nishadi/ppaas-artifact-converter/src/main/resources/template/network-partitions/gce/network-partition-1.json
+java.io.FileNotFoundException: /home/nishadi/ppaas-artifact-converter/src/main/resources/template/network-partitions/gce/network-partition-1.json
+	at java.io.FileInputStream.open(Native Method)
+	at java.io.FileInputStream.<init>(FileInputStream.java:146)
+	at java.io.FileInputStream.<init>(FileInputStream.java:101)
+	at java.io.FileReader.<init>(FileReader.java:58)
+	at org.wso2.ppaas.tools.artifactmigration.loader.TemplateLoader.fetchTemplate(TemplateLoader.java:57)
+	at org.wso2.ppaas.tools.artifactmigration.Transformation.transformNetworkPartitionList(Transformation.java:123)
+	at org.wso2.ppaas.tools.artifactmigration.ArtifactConverter.main(ArtifactConverter.java:36)
```

Comment:
We don't need to add the log files to the git.Please configure them with the .gitignore